CHANGELOG.md

```diff
@@ -10,6 +10,7 @@
 * [CHANGE] Ingester/Distributor: renamed the experimental `max_cost_attribution_cardinality_per_user` config to `max_cost_attribution_cardinality`. #11092
 * [CHANGE] Frontend: The subquery spin-off feature is now enabled with `-query-frontend.subquery-spin-off-enabled=true` instead of `-query-frontend.instant-queries-with-subquery-spin-off=.*` #11153
 * [FEATURE] Distributor: experimental support for Prometheus Remote Write 2.0 protocol. Limitations: created timestamp is ignored, native histograms with custom buckets are rejected, per series metadata is merged on metric family level automatically, ingestion might fail if client sends ProtoBuf fields out of order. #11100 #11101
+* [ENHANCEMENT] Distributor: Report artificialDelay on ServerTiming header for push requests. #11178
```

Comment:
This is a testing feature that we haven't documented so far. I would keep it undocumented.

```suggestion
```